doc/tutorial.rst

```diff
@@ -142,7 +142,7 @@ of the collections in our database:
 
 .. doctest::
 
-  >>> db.collection_names(include_system_collections=False)
+  >>> db.list_collection_names(include_system_collections=False)
```

Comment:
Thanks for finding this issue! It also made me realize that we didn't properly support  the [Enumerate Collections Specification](https://github.com/mongodb/specifications/blob/master/source/enumerate-collections.rst). I fixed that by adding support for `filter` argument in https://jira.mongodb.org/browse/PYTHON-1784.

Can you replace the `include_system_collections` parameter with a `filter` that only matches non-system collections, like this:
```
>>> db.list_collection_names(filter={"name": {"$regex": r"^(?!system\.)"}})
[u'posts']
```

Reply:
I just realized that for this doctest it's fine to just remove the `include_system_collections=False`. We only run the doctests on 4.0+ so we don't need to worry about system collections like (`system.indexes`) showing up in the results.